modules/users/devices.nix

```diff
@@ -0,0 +1,26 @@
+let
+  pixel8Keys = [
+    "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOPHHU+BO8E/+Mn98QFbkbz9mYm5wNUTBYGOO8EMGg7o pixel8"
```

Comment:
who owns this?

Reply:
The key? It's on the phone that's lying in the hardware lab.

Reply:
So the key ssh back to graham?

Reply:
Yeah, the phone uses this key to connect to graham. The authorized_keys on the phone are managed manually.

Reply:
Okay, we should remove this key eventually if it's no longer needed for evaluation.
Students can also use ssh agent forwarding to access machines from other machines.
Otherwise removing old users no longer is effective.